src/main/kotlin/com/rooplor/classcraftbackend/repositories/ClassroomRepository.kt

```diff
@@ -14,4 +14,9 @@ interface ClassroomRepository : MongoRepository<Classroom, String> {
     fun findByIsPublishedTrueOrderByCreatedWhen(): List<Classroom>
 
     fun findByOwner(owners: String): List<Classroom>
+
+    fun findByIsPublishedTrueAndTitleContainingIgnoreCaseOrDetailsContainingIgnoreCaseAndIsPublishedTrue(
```

Comment:
Because it dose not had `()` . If I check ispublish only one something like this `findByIsPublishedTrueAndTitleContainingIgnoreCaseOrDetailsContainingIgnoreCase` when it query it will check something like (IsPublish == true && title contain ("keyword") || detail contain ("keyword"))
that why I need to check twice.

Reply:
Can you example raw query with check isPublished in twice?

Reply:
Okie I see